scRNA-seq/01-normalizing_scRNA-seq.Rmd

```diff
@@ -61,28 +62,222 @@ and could use these same steps to filter and normalize the data.
   
 ```{r Import Data}
 # Read in the data
-sc_data <- readr::read_tsv(file.path("data", "darmanis_counts.tsv"), 
-                           progress = FALSE) %>%
-  as.data.frame() %>%
-  # We will need our genes as the rownames for setting up our data with Seurat
-  tibble::column_to_rownames("gene") 
+sc_data <- readr::read_tsv(file.path("data", "unfiltered_darmanis_counts.tsv"), 
+                           progress = FALSE)
+
+# Read in the metadata
+sc_metadata <- readr::read_tsv(file.path("data", "unfiltered_darmanis_metadata.tsv"), 
+                               progress = FALSE)
+```
+
+## Filter the counts matrix
+  
+Single-cell RNA-seq experiments are generally much larger dimensions than bulk 
+RNA-seq, since every individual cell is being measured. 
+However, due to the high risk of technical artifacts, we don't necessarily want 
+to keep the data for all the cells and genes. 
+
+There are different approaches to filtering your data. 
+In an ideal world, all RNA-seq data would come with the percent mapped reads 
```

Comment:
I want people to be aware of this 1) So they report their own percent mapped reads of their data and 2) so that they keep it in mind when processing their own data.

Reply:
To me, it may make more sense to tackle this when they are either processing bulk RNA-seq data with Salmon _or_ when they are processing data with Alevin (does Alevin report this information?). Here it seems out of left-field.

Reply:
Will remove this then